docs/digger/installation/run-digger-installer.asciidoc

```diff
@@ -1,22 +1,22 @@
-== Run the Digger Installer
+== Running the Digger Installer
```

Comment:
Let's remove `the` here. sounds a bit funny.

Reply:
resolved